gulpfile.js

```diff
@@ -91,7 +91,7 @@ function lint(done) {
     '!plugins/**/node_modules/**',
     './*.js'
   ], { base: './' })
-    .pipe(eslint({ fix: !argv.nolintfix, quiet: !(typeof argv.lintWarnings === 'boolean' ? argv.lintWarnings : true) }))
+    .pipe(eslint({ fix: !argv.nolintfix, quiet: argv.lintWarnings === 'false' }))
```

Comment:
This setting wasn't working as `lintWarnings` was always of type `string`.

```bash
gulp lint --lintWarnings=false
```

now works as expected

Reply:
it's a boolean if you use `--no-lint-warnings`.